{"uuid": "7d2aac65-5395-46c5-9a43-df2605da616c", "children": ["e8fddf28-cd9f-418e-aebb-933d0966d358"], "befores": [{"name": "token", "status": "broken", "statusDetails": {"message": "AttributeError: 'Response' object has no attribute 'get'\n", "trace": "  File \"/Users/xiaotuji/PycharmProjects/Meikefresh/.venv/lib/python3.8/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Users/xiaotuji/PycharmProjects/Meikefresh/.venv/lib/python3.8/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/Users/xiaotuji/PycharmProjects/Meikefresh/.venv/lib/python3.8/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/xiaotuji/PycharmProjects/Meikefresh/.venv/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/xiaotuji/PycharmProjects/Meikefresh/.venv/lib/python3.8/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/Users/xiaotuji/PycharmProjects/Meikefresh/testcase/conftest.py\", line 23, in token\n    token = response.get('token')\n"}, "start": 1740037540010, "stop": 1740037540216}], "start": 1740037540010, "stop": 1740037540283}